docs/misc/changelog.rst

```diff
@@ -46,6 +46,7 @@ Others:
 - Add upper bound for Tensorflow version (<2.0.0).
 - Refactored test to remove duplicated code
 - Add pull request template
+- Replaced redundant code in load_results
```

Comment:
Please put your nickname too (and at the bottom).
Entries without are from maintainers.